assets/src/edit-story/components/helpCenter/quickTip/index.js

```diff
@@ -68,11 +70,46 @@ const Paragraph = styled(Text)`
   }
 `;
 
+const DoneContainer = styled.div`
+  ${themeHelpers.centerContent}
+  height: 135px;
+  width: 100%;
+`;
+
+const CheckmarkContainer = styled.div`
+  ${themeHelpers.centerContent}
+  border-radius: ${dsTheme.borders.radius.round};
```

Comment:
Could we use the design system theme here directly? or does it not exist on that object
```suggestion
  border-radius: ${({ theme }) => theme.borders.radius.round};
```